fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/data/DepositProductDataValidator.java

```diff
@@ -115,7 +113,7 @@ public void validateForFixedDepositCreate(final String json) {
 
         final JsonElement element = this.fromApiJsonHelper.parse(json);
 
-        validateDepositDetailForCreate(element, this.fromApiJsonHelper, baseDataValidator);
+        validateDepositDetailForCreate(element, this.fromApiJsonHelper, baseDataValidator, DepositAccountType.RECURRING_DEPOSIT);
```

Comment:
Validation of Fixed deposit is the method name but here you are providing Recurring deposit. Why?

Reply:
Done!